docs/sources/get-started/ride-share-tutorial.md

```diff
@@ -0,0 +1,297 @@
+---
+description: Learn how to get started with Pyroscope using a simple Ride share app.
+menuTitle: Ride share tutorial
+title: Ride share tutorial with Pyroscope
+weight: 250
+killercoda:
+  title: Ride share tutorial
+  description: Learn how to get started with Pyroscope using a simple Ride share app.
+  details:
+      intro:
+         foreground: docker-compose-update.sh
+  backend:
+    backend:
+    imageid: ubuntu
+---
+
+<!-- INTERACTIVE page intro.md START -->
+
+# Ride share tutorial with Pyroscope
+
+This tutorial demonstrates a basic use case of Pyroscope by profiling a "Ride Share" application.
+In this example, you learn:
+
+- How an application is instrumented with Pyroscope, including techniques for dynamically tagging functions.
+- How to view the resulting profile data in the Pyroscope UI.
+- How to integrate Pyroscope with Grafana to visualize the profile data.
+
+<!-- INTERACTIVE ignore START -->
+
+## Before you begin
+
+You need to have the following prerequisites to complete this tutorial:
+- Git
+- [Docker](https://docs.docker.com/compose/install/)
+- The Docker Compose plugin (included with Docker Desktop)
+
+{{< admonition type="tip" >}}
+Try this tutorial in an interactive learning environment: [Ride share tutorial with Pyroscope](https://killercoda.com/grafana-labs/course/pyroscope/ride-share-tutorial).
+
+It's a fully configured environment with all the dependencies installed.
+
+Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
+{{< /admonition >}}
+
+<!-- INTERACTIVE ignore END -->
+
+## Background
+
+{{< figure max-width="100%" src="/media/docs/pyroscope/ride-share-demo.gif" caption="Getting started sample application" alt="Getting started sample application" >}}
```

Comment:
We usually put images after the text, so I'll have two suggestions to move the figure. 
```suggestion
```